code/modules/hydroponics/hydroponics.dm

```diff
@@ -75,6 +75,7 @@
 	plant_hud_set_water()
 
 /obj/machinery/hydroponics/Destroy()
+	remove_from_all_data_huds()
```

Comment:
Did I miss something with my Hydro HUD? I need to know any and all mistakes I may and may not have made please